velero-plugins/imagestream/shared.go

```diff
@@ -135,23 +134,3 @@ func getSecretKeyRefData(secretKeyRef *corev1.SecretKeySelector, namespace strin
 	}
 	return secret.Data[secretKeyRef.Key], nil
 }
-
-// writes data to file. If file exists, overwrites it.
-func saveDataToFile(data []byte, path string) error {
-	// delete path if it exists
-	if _, err := os.Stat(path); err == nil {
-		if err := os.Remove(path); err != nil {
-			return err
-		}
-	}
-	file, err := os.Create(path)
-	if err != nil {
-		return err
-	}
-	defer file.Close()
-	_, err = file.Write(data)
-	if err != nil {
-		return err
-	}
-	return nil
-}
```

Comment:
Inheriting creds in velero pod written by velero https://github.com/vmware-tanzu/velero/blob/5afe837f76aea4dd59b1bf2792e7802d4966f0a7/internal/credentials/file_store.go#L72

So we don't need to write our own cred file.